modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookManagementToolbarDisplayContext.java

```diff
@@ -40,13 +47,17 @@ public class StyleBookManagementToolbarDisplayContext
 	public StyleBookManagementToolbarDisplayContext(
 		HttpServletRequest httpServletRequest,
 		LiferayPortletRequest liferayPortletRequest,
-		LiferayPortletResponse liferayPortletResponse,
+		LiferayPortletResponse liferayPortletResponse, CETManager cetManager,
```

Comment:
Sort parameters, the first should be `cetManager`. 

I thought that could be a pattern, to have the requests at the beginning, but doesn't seem to be (example: ChannelManagementToolbarDisplayContext).